pgoutput_parser/update.py

```diff
@@ -29,10 +29,13 @@ def decode_update_message(self) -> dict:
             logger.debug(f'Relation ID: {relation_id}')
             logger.debug(f'Old tuple: {old_tuple}')
             logger.debug(f'New tuple: {new_tuple}')
-
             logger.debug(f'Old tuple values: {old_tuple_values}')
             logger.debug(f'New tuple values: {new_tuple_values}')
 
+            if not new_tuple_values or 'id' not in new_tuple_values:
```

Comment:
what case is this? can you also add comment to the code

Reply:
can update this according to latest disucssion, can change `'id': new_tuple_values.get('id') or old_tuple_values.get('id'),`